### PR TITLE
Merge flappy and flappy_birds

### PR DESCRIPTION
**Pull request by Dimi20cen**
Merge flappy with flappy-birds

- Add up-arrow key functionality for bird movement
- Refine bird rotation for smoother gameplay
- Implement a moving background for enhanced visuals
- Incorporate flapping wings animation
- Adjust the starting point of bottom pipes for better visuals
- Enhance collision detection, using masks, for increased precision
- Introduce a sound effect upon collision, with the bird falling to the ground
- Tweak the points sound effect timing for improved feedback
- Update the game font and add an outline, to resemble the original Flappy Bird
- Replace the existing Sugargame file with the latest version
- Add flake8 file

@chimosky @sourabhaa

### CODE REVIEW

File: bird.py [hunk 1]
```diff
@@ -3,52 +3,67 @@
...
-        self.mVel = vel * self.factor
-        self.count = 20
-        self.setImage(bird_u)
+        self.velocity_limit = -20 * self.factor ** 1.7
+        self.acceleration = self.factor ** 0.6
+        self.images = []
+        self.index = 0
+        self.counter = 0
+        self.count_flap = 0
+        for num in range(0, 3):
```
Comment (chimosky) [inline bird.py]: You don't need to specify 0 here.

File: main.py [hunk 2]
```diff
@@ -211,9 +217,14 @@ def run(self):
...
             pygame.display.flip()
-            self.clock.tick(30)
+            self.clock.tick(FPS)
 
 
 if __name__ == "__main__":
     g = Flappy()
+    # GAME_SIZE = (1920, 900)
+    # GAME_SIZE = (1024, 768)
+    # GAME_SIZE = (684, 600)
```
Comment (chimosky) [inline main.py]: You can delete these.

**Comment by chimosky** [comment]
@Dimi20cen I opened a PR in your master branch, couldn't push here as you're working on the master branch.

File: bird.py (hunk 1 above)
Comment (Dimi20cen) [reply bird.py]: Do you suggest I shouldn't initialize index, counter and count_flap?

File: main.py (hunk 2 above)
Comment (Dimi20cen) [reply main.py]: I left them in for more convient testing outside of sugar env, is it not good practice?

File: bird.py (hunk 1 above)
Comment (chimosky) [reply bird.py]: Not at all, I'm saying your range can be `range(3)` as you're not using index, counter or count_flap in your range loop.

File: main.py (hunk 2 above)
Comment (chimosky) [reply main.py]: As they're for testing, they shouldn't be included in your commit as you're done with testing and have settled on what to use.

**Comment by sourabhaa** [comment]
tested, I'm able to see the toolbar,
Screenshot of _Flappy Activity_

**Comment by jriyyya** [comment]
Tested [dec589d](https://github.com/sugarlabs/flappy/pull/18/commits/dec589d756c025c25a74736665cd20b79039311f), Toolbar is visible and functional to me

**Comment by chimosky** [comment]
Tested dec589d and the toolbar shows, I'll compare with flappy birds before merging.

**Comment by chimosky** [comment]
I compared with flappy birds activity and they're a few things left;

- [ ] sound; Flappy birds have sound from start of the game till the end of the game.
- [ ] Game speed; flappy seems to have a very slow game speed compared to flappy birds and it would be great to increase the game speed as it's more interesting.

**Comment by Dimi20cen** [comment]
> sound; Flappy birds have sound from start of the game till the end of the game.

Could you clarify? I think the merged activity and flappy-birds have, identical sounds.

> Game speed; flappy seems to have a very slow game speed compared to flappy birds and it would be great to increase the game speed as it's more interesting.

While x-speed is slower than flappy's, I aimed to mimic the x-speed of the original implementation of the game flappy birds, trusting that they'd found the sweet spot.

**Comment by chimosky** [comment]
> Could you clarify? I think the merged activity and flappy-birds have, identical sounds.

There are no sounds when I test your changes, @jriyyya @sourabhaa was the sound working fine when you tested?

> While x-speed is slower than flappy's, I aimed to mimic the x-speed of the original implementation of the game flappy birds, trusting that they'd found the sweet spot.

Okay.

**Comment by Dimi20cen** [comment]
Sound should be fixed @chimosky.

**Comment by chimosky** [comment]
Tested c6062e1, sound now works as expected. Thanks!

**Pull request merged**